### DISCUSSION TITLE
Show empty windows.

### DISCUSSION THREAD

**Discussion by h0r4ci0** [polybar/polybar#3046] [Q&A] [answered]
Hey y'all!
I was trying to view empty windows on the polybar, however I haven't made any progress. I can only view active and occupied windows.

> [!NOTE]
> Currently, I'm using i3 as WM.

**Workspaces module preview**:

![image](https://github.com/polybar/polybar/assets/87150874/089c7c7c-afc8-48e1-8403-2e2acc0cc229)

**Workspaces module code**:

```sh
[module/workspaces]
type = internal/xworkspaces

pin-workspaces = true
enable-click = false
enable-scroll = false

format = <label-state>
label-monitor = %name%

label-active-foreground = ${colors.foreground}
label-active-background = ${colors.background}
label-active-padding = 4
label-active = 󰕗

label-occupied-foreground = ${colors.foreground}
label-occupied-background = ${colors.background}
label-occupied-padding = 4
label-occupied = 󰊠

label-urgent-foreground = ${colors.foreground}
label-urgent-background = ${colors.background}
label-urgent-padding = 4
label-urgent = 

label-empty-foreground = ${colors.foreground}
label-empty-background = ${colors.background}
label-empty-padding = 4
label-empty = 󱙝
```

Thanks for your time!

**Comment by patrick96** [comment]
empty workspaces don't exist in i3. A workspace id destroyed once it becomes empty (and is not visible)

**Reply by h0r4ci0** [reply]
So, there's no way to represent this empty workspaces in the polybar?

**Reply by patrick96** [reply, marked as answer]
There is not
